tests/controller-test.js

```diff
@@ -217,11 +226,16 @@ OO = {
     Html5Skin.subscribeBasicPlaybackEvents.call(controllerMock);
     Html5Skin.externalPluginSubscription.call(controllerMock);
 
-    // test player state
-    Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', 1, {skin:{config:{}}}, {});
+    //test player state
+    var tempSkin = controllerMock.skin;
+    Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', elementId, {skin:{config:{}}}, {});
+    Html5Skin.loadConfigData.call(controllerMock, {skin:{config:{}}}, {});
+    Html5Skin.createPluginElements.call(controllerMock);
+    controllerMock.skin = tempSkin; //reset skin, onPlayerCreated updates skin
```

Comment:
I think that tempSkin would point to the same object as controllerMock.skin, so we'd probably need to clone it in order for this to work as a reset.